pkg/types/clustermetadata.go

```diff
@@ -20,6 +21,13 @@ type ClusterAWSPlatformMetadata struct {
 	Identifier map[string]string `json:"identifier"`
 }
 
+// ClusterOpenStackPlatformMetadata contains OpenStack metadata.
+type ClusterOpenStackPlatformMetadata struct {
+	Region string `json:"region"`
+	// Most OpenStack resources are tagged with these tags as identifier.
+	Identifier map[string]string `json:"identifier"`
```

Comment:
Can we use `UserTags` for [consistency][1]?

[1]: https://github.com/openshift/installer/blob/0284d4e9a099dd131b2730c61cc0ac3876907290/pkg/types/installconfig.go#L93-L94

Reply:
@wking this is `clustermetadata` used to identify cluster after done. Not `installconfig`.